packages/web-runtime/src/pages/accessDenied.vue

```diff
@@ -3,77 +3,61 @@
     class="oc-login oc-height-viewport"
     :style="{ backgroundImage: 'url(' + backgroundImg + ')' }"
   >
-    <h1 class="oc-invisible-sr">{{ pageTitle }}</h1>
-    <div class="oc-login-card oc-position-center">
-      <img class="oc-login-logo" :src="logoImg" alt="" :aria-hidden="true" />
-      <div class="oc-login-card-body oc-width-large">
-        <h3 class="oc-login-card-title">
-          <span v-translate>Login Error</span>
-        </h3>
-        <h4 v-translate class="oc-mb-m">Your user session is invalid or has expired.</h4>
-        <div v-translate class="oc-mb-m">
-          If you like to login with a different user please proceed to
-          <router-link id="exitAnchor" :to="{ name: 'login' }">exit</router-link>
+    <h1 class="oc-invisible-sr" v-text="pageTitle" />
+    <div class="oc-login-card-wrapper oc-height-viewport oc-flex oc-flex-center oc-flex-middle">
+      <div class="oc-login-card">
+        <img class="oc-login-logo" :src="logoImg" alt="" :aria-hidden="true" />
+        <div class="oc-login-card-body oc-width-large">
```

Comment:
@JammingBen this is already width-large (and was before I changed things) so adding medium to the parent wouldn't make sense?

Reply:
Ah sorry, then using `oc-width-medium` here and on the button would be the way to go:

![image](https://user-images.githubusercontent.com/50302941/194544828-5d7ac7f1-cf40-41d2-9507-7b1928d525b2.png)

Reply:
Alright let's go then, just a sec